ui/app/src/components/Navigation/PathNavigator/styles.ts

```diff
@@ -76,6 +76,10 @@ export const useStyles = makeStyles(
         padding: 0,
         flexDirection: 'column'
       },
+      childrenRail: {
+        marginLeft: 10,
+        borderLeft: `3px solid ${palette.gray.light2}`
```

Comment:
Does the color work well on both theme modes? 
Try `theme.palette.paper.default` or divider color